core/import-shearwater.c

```diff
@@ -394,7 +394,7 @@ static int shearwater_cloud_dive(void *param, int columns, char **data, char **c
 		state->cur_dive->dc.maxdepth.mm = state->metric ? lrint(strtod_flags(data[6], NULL, 0) * 1000) : feet_to_mm(strtod_flags(data[6], NULL, 0));
 
 	if (data[7])
-		state->cur_dive->dc.duration.seconds = atoi(data[7]) * 60;
+		state->cur_dive->dc.duration.seconds = atoi(data[7]);
```

Comment:
Since all this used to work... is there any version information in their cloud database? Anything that would help alert us to such random changes?

Reply:
I am actually wondering, if this was calculated based on samples. And it was only brought visible by the dive profile code not working in Windows.